app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/HiddenHealthRecordAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package ca.bc.gov.bchealth.ui.healthrecord.individual
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.databinding.ItemLoginForHiddenRecordsBinding
+
+class HiddenHealthRecordAdapter(
+    private val loginClickListener: ItemClickListener,
+    private val context: Context
```

Comment:
remove context from the constructor,
You can use context from item view in Adapter

Reply:
Done